src/lib/js/components/controls/index.js

```diff
@@ -379,7 +375,14 @@ export class Controls {
     this.container = container
     this.groupOrder = unique(groupOrder.concat(['common', 'html', 'layout']))
     this.options = options
-    return Promise.all(this.registerControls([...defaultElements, ...elements]))
+
+    const layoutControls = await import('./layout/index.js')
```

Comment:
using dynamic import to prevent circular dependency found when writing tests for controls